TRAMS-API/Models/Downstream/D365/GetProjectAcademiesResponseModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿//using API.Models.D365;
+//using Newtonsoft.Json;
+//using System;
+//using System.Collections.Generic;
+//using System.Linq;
+//using System.Threading.Tasks;
+
+//namespace API.Models.Downstream.D365
+//{
+//    public class GetProjectAcademiesResponseModel : BaseD365Model
+//    {
+//        [JsonProperty("sip_academytransfersprojectacademyid")]
+//        public Guid AcademyTransfersProjectAcademyId { get; set; }
+
+//        [JsonProperty("_sip_atprojectid_value")]
+//        public Guid ProjectId { get; set; }
+
+//        [JsonProperty("_sip_academyid_value")]
+//        public Guid AcademyId { get; set; }
+
+//        [JsonProperty("_sip_academyid_value@OData.Community.Display.V1.FormattedValue")]
+//        public string AcademyName { get; set; }
+
+//        [JsonProperty("sip_esfainterventionreason")]
+//        public string EsfaInterventionReasons { get; set; }
+
+//        [JsonProperty("sip_rddorrscinterventionreasonsexplanation")]
+//        public string EsfaInterventionReasonsExplained { get; set; }
+
+//        [JsonProperty("sip_rddorrscinterventionreasons")]
+//        public string RddOrRscInterventionReasons { get; set; }
+
+//        [JsonProperty("sip_esfainterventionreasonexplanation")]
+//        public string RddOrRscInterventionReasonsExplained { get; set; }
```

Comment:
Question / Potential Error: Are the json property names for EsfaInterventionReasonsExplained and RddOrRscInterventionReasonsExplained the wrong way round?  

At present the Esfa one points to the rddorrsc field in CRM and teh RddOrRsc one points to the esfa field in CRM.

I'm convinced you've put this here on purpose to test my eyesight :)

Reply:
Class is commented out and not used. Will remove